.packit.yaml

```
@@ -11,7 +11,7 @@ jobs:
       job: copr_build
       trigger: pull_request
       targets:
-          - fedora-all-aarch64
+          - epel-7-x86_64
```

I think there is no reason not to test all EPEL buildroots.

```suggestion
          - epel-all-aarch64
          - epel-all-ppc64le
          - epel-all-s390x
          - epel-all-x86_64
          - fedora-all-aarch64
```

---

Fixed in ceeb4cfeadb52016c15cf0f21c9c256cef54cbda.

---

I guess the reason is that `nss-pkcs11-devel` is not in CRB.